config/_default/menus/menus.en.toml

```diff
@@ -56,18 +56,18 @@
     weight = 400
 [[footer]]
     parent = "column1"
-    name = "Documentation"
-    url = "/docs/"
+    name = "Download"
+    url = "/docs/install/"
     weight = 410
 [[footer]]
     parent = "column1"
-    name = "Language Guide"
-    url = "/docs/language-guide/"
+    name = "Get started"
+    url = "/docs/get-started/"
     weight = 411
 [[footer]]
     parent = "column1"
-    name = "Playground"
-    url = "/play/"
+    name = "Reference Documentation"
+    url = "/docs/references/"
```

Comment:
right now, the only Reference Documentation is the Language Specification. You can create a placeholder single page with a TODO block for now, but this will be another large piece of content similar to language guide for template/layout. 

docs/references/spec/

Reply:
the language specification is quite lengthy, so the sitemap is being updated to call it a "future book" that will live under docs. But a placeholder page works for now.